# Basic-Bioinformatics-Tools
The most well-organised, documented, and utilized bioinformatics tools. The foundations if you will, of analysis. 

Perhaps the most frustating aspect of bioinformatics is the huge variety and spectrum of tools out there. With so many unknowns I propose a list of tools, for all common purposes, to more or less standardize the field (wish me good luck). 

Frankly, an incredible amount of tools are published for someone's PhD or Masters thesis, and promptly forgotten. Yet Google never forgets. Searching for "phylogeny bioinformatics tools" yields answers on forums such as ResearchGate, BioStars, random publications of little known tools, OMICTools and other collation sites -- essentially the whole gambit.

I really feel that only certain tools need to be used for the issues of at least basic reproducibility. These seem mature enough and versatile enough to accomplish most basic tasks or workflows. The emphasis is on open source tools and tools available on github. TRF, USEARCH, and other tools are great, but not having access to the source on version control (TANTAN) becomes a big deal when something goes down.  

## SAM/BAM 

Samtools
Bamtools
BWA

## RNA-seq

STAR
Cufflinks
Tophat

## Ortholog Analysis

Orthofinder
BUSCO

## Phylogenetics

ETE

## Alignment

Diamond
BLAT
BLAST
GHOSTX

## Metagenomics

VSEARCH

Very hesitant to include BLAST here as it is universal at this point, but better options exist.


